public/js/initiate.js

```diff
@@ -31,15 +34,16 @@ window.onload =  function() {
     //
     // Create <a-image> element
     //
-    var image = document.createElement("a-image");
-    image.setAttribute('src', 'img/' + letter + '.png');
-    image.setAttribute('scale', '1.5 1.5 1.5');
+    var image_box = document.createElement("a-box");
+    image_box.setAttribute('src', 'img/' + letter + '.png');
+    image_box.setAttribute('scale', '1.5 1.5 1.5');
+    // image_box.setAttribute('position', position);
```

Comment:
Uncomment this line on your development to see the 360 degree revolving of all the letters, this will ensure the 360 degree space is covered.